CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#9235](https://github.com/cosmos/cosmos-sdk/pull/9235) CreateMembershipProof/CreateNonMembershipProof now returns an error
 if input key is empty, or input data contains empty key.
 * [\#9314](https://github.com/cosmos/cosmos-sdk/pull/9314) Update Rosetta SDK to upstream's latest release.
+* [\#9371](https://github.com/cosmos/cosmos-sdk/pull/9371) Non-zero default fees/Server will error if there's an empty value for min-gas-price
```

Comment:
Is this a breaking change? I would actually say yes:
- before: if the app.toml#min-gas-price is empty, it would default to 0
- after: if the app.toml#min-gas-price is empty, the node will error

But I still think this could go in 0.43.